com.unity.shadergraph/Editor/ShaderGraphPreferences.cs

```diff
@@ -71,13 +79,21 @@ static void OpenGUI()
                 autoAddRemoveBlocks = autoAddRemoveBlocksValue;
             }
 
+            EditorGUI.BeginChangeCheck();
+            var allowDeprecatedBehaviorsValue = EditorGUILayout.Toggle("Allow the use of deprecated versions of properties and nodes", allowDeprecatedBehaviors);
```

Comment:
needs better display name

Reply:
"Enable deprecated nodes" - I think that covers it as long as the docs are clear it includes properties too.

Reply:
Agreed